src/main/java/mate/academy/spring/controller/MockDataController.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDateTime;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import mate.academy.spring.service.MovieSessionService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MockDataController {
```

Comment:
```suggestion
public class DataInitializer {
```
it's not a controller now